sentry_sdk/crons/_decorator.py

```diff
@@ -0,0 +1,29 @@
+from functools import wraps
+from inspect import iscoroutinefunction
+
+from sentry_sdk._types import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from typing import Any, Callable
+
+
+class _MonitorMixin:
```

Comment:
[nitpick]

```suggestion
class MonitorMixin:
```

I would suggest removing the underscore here (and also in the Python 2 decorator) so that VSCode does not mis-recognize this as an unused variable. The underscore in the filename should be sufficient to indicate that this file's contents are private API.